components/WithdrawLiquidity/WithdrawLiquidityController.tsx

```diff
@@ -180,7 +186,10 @@ const WithdrawLiquidity = () => {
   }, [provider])
 
   function pretty(num: number) {
-    return parseFloat(ethers.utils.formatEther(num.toString()).toString()).toFixed(2)
+    return ethers.utils
+      .formatEther(num.toString())
+      .toString()
+      ?.match(/^-?\d+(?:\.\d{0,2})?/)![0]
```

Comment:
What's this line do?

Reply:
This line is restricting the value to two decimal point without rounding the decimal point. Like earlier when the value was 9.8645613 the program rounds the value to 10.00 but now with this code it will convert the value to two decimal point and will not round it. It will return 9.86

Reply:
So the value is still 9.8645613 but display is 9.86? Can you check how the other plaforms do?

Reply:
Yes the value is still the original but we are only viewing first three characters from there. I will look what other platform are doing as well.